parquet-column/src/main/java/org/apache/parquet/column/values/boundedint/BoundedIntValuesWriter.java

```diff
@@ -101,6 +104,16 @@ public void reset() {
   }
 
   @Override
+  public void close() {
+    currentValue = -1;
+    currentValueCt = -1;
+    currentValueIsRepeated = false;
+    thereIsABufferedValue = false;
+    isFirst = true;
+    bitWriter.close();
```

Comment:
I'm not sure we need to do all of that since we won't be able to write to it once the bitWriter is closed.